cinema/urls.py

```diff
@@ -16,6 +18,8 @@
 router.register("movies", MovieViewSet)
 router.register("movie_sessions", MovieSessionViewSet)
 
+router.register("orders", OrdersViewSet)
```

Comment:
The `OrdersViewSet` is registered correctly, ensuring that the order-related endpoints are available in the API.